data/chains/katana.json

```diff
@@ -0,0 +1,27 @@
+{
+  "alias": "katana",
+  "decimals": 18,
+  "explorer": {
+    "api": {
+      "key": {
+        "required": false
+      },
+      "url": "https://explorer.katanarpc.com/api"
+    },
+    "browserUrl": "https://explorer.katanarpc.com/"
+  },
+  "id": "747474",
+  "name": "Katana",
+  "providers": [
+    {
+      "alias": "default",
```

Comment:
I was about to suggesting to add https://rpc.katanarpc.com introduced in https://github.com/ToniRamirezM/chains/blob/389f820c68fb361ecc8b5637d1050606fdcf619a/_data/chains/eip155-747474.json#L4 but checked that it isn't working. So it's okay to go with these providers, maybe we can consider to run node for the chain.